ranger/gui/ui.py

```diff
@@ -485,6 +491,33 @@ def get_pager(self):
     def _get_viewmode(self):
         return self._viewmode
 
+    def _set_titlebar(self, value):
```

Comment:
Can't you work with a default value here? `... , value=True):` And then simplify the body to test on a boolean rather than these `ALLOWED_STATUS` strings?

Reply:
will do.